conjur/wrapper/keystore_adapter.py

```diff
@@ -65,15 +65,17 @@ def get_keyring_name(cls):
         """
         return keyring.get_keyring().name
 
-    # TODO add a more robust check
     @classmethod
     def is_keyring_accessible(cls):
         """
         Method to check if the keyring is accessible
         """
-        # TODO investigate other errors that would make a keyring not accessible
         try:
-            # Send a dummy request to test if the keyring is accessible
+            # The following sends a dummy request to test if the system's keyring is
+            # accessible. 'get_password' returns None if the password does not exist.
+            # If get_password raises an error, then we can infer that the keyring has
+            # not be configured correctly and not available for use and return false
```

Comment:
If get_password raises an error, then we can infer that the keyring has
not **been** configured correctly and not available for use and return false

there is A and B and C in the above sentence